Wikipedia/UI-V5/WMFLocationManager.m

```diff
@@ -6,8 +6,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static CLLocationDistance WMFMinimumDistanceBeforeUpdatingLocation = 1.0; //meters before we update location
-
 @interface WMFLocationManager ()<CLLocationManagerDelegate>
```

Comment:
There was some thrashing here, but the latest commit(s) should have comments explaining the restored legacy behavior:

- Update location every 1m (i.e. `distanceFilter = 1`)
- Re-fetch location search results every 500m (see check in `-[WMFNearbyViewModel fetchTitlesForLocation:]`)